src/app/features/data-import/data-import.service.ts

```diff
@@ -134,11 +140,29 @@ export class DataImportService {
       return Entity.createPrefixedId(importMeta.entityType, value);
     } else if (importMeta.dateFormat && this.dateDataTypes.includes(dataType)) {
       return this.transform2Date(value, importMeta.dateFormat);
+    } else if (this.arrayDataTyps.includes(dataType)) {
```

Comment:
do we need to stack this recursively in a way so that array items might also be parsed with transform2Date?

Reply:
Generally yes. I would however postpone this to when there is actually a use-case for this. At the moment our app is not supporting arrays of dates anyways and there are no other data types that are transformed during import.